pandas/core/series.py

```diff
@@ -5313,6 +5314,8 @@ def _arith_method(self, other, op):
 
         lvalues = self._values
         rvalues = extract_array(other, extract_numpy=True, extract_range=True)
+        lvalues = ensure_wrapped_if_datetimelike(lvalues)
+        rvalues = ensure_wrapped_if_datetimelike(rvalues)
```

Comment:
Here it can in theory be removed for `lvalues`, I think

Reply:
i think you're right, yah